contracts/hardhat.config.ts

```diff
@@ -86,10 +87,10 @@ const config: HardhatUserConfig = {
     timeout: 200000, // 200 seconds max for running tests
   },
   etherscan: {
-    // apiKey: {
-    //   polygonMumbai: POLYGONSCAN_KEY,
-    // },
-    apiKey: ETHERSCAN_API_KEY ? ETHERSCAN_API_KEY : '',
+    apiKey: {
+      polygonMumbai: POLYGONSCAN_KEY,
+    },
+    // apiKey: ETHERSCAN_API_KEY ? ETHERSCAN_API_KEY : '',
```

Comment:
maybe we are able to add for both networks?

Reply:
done